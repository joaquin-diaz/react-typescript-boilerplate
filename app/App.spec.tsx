import * as React from 'react';
import { mount } from 'enzyme';

import App from './App';

describe('<App>', () => {
  it('Should render', () => {
    const wrapper = mount(<App />);
    expect(wrapper).toMatchSnapshot();
  });
});
