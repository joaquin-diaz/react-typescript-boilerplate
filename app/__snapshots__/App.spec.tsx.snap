// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App> Should render 1`] = `
<App>
  <div>
    React App
  </div>
</App>
`;
